full/Dockerfile

```diff
@@ -92,6 +92,9 @@ RUN echo "deb https://dl.yarnpkg.com/debian/ stable main" | tee /etc/apt/sources
     && apt-get clean && rm -rf /var/cache/apt/* && rm -rf /var/lib/apt/lists/* && rm -rf /tmp/* \
     && echo Installing Yarn newer than 1.10.1
 
+# Instal JS/TS
```

Comment:
should be before `USER root` and the comment has a type

Reply:
fixed